requirements.txt

```diff
@@ -1,3 +1,3 @@
-git+https://github.com/astronomy-commons/hipscat.git@main
-git+https://github.com/astronomy-commons/hipscat-import.git@main
-git+https://github.com/astronomy-commons/lsdb.git@main
\ No newline at end of file
+git+https://github.com/astronomy-commons/hipscat.git@hats
+git+https://github.com/astronomy-commons/hipscat-import.git@hats
```

Comment:
Branch change preparatory to repo change, I assume

Reply:
Yes. Repo renaming coming .... eventually.